### DISCUSSION TITLE
Is swr supported in all browsers?

### DISCUSSION THREAD

**Discussion by jross53** [vercel/swr#483] [General] [answered]
I noticed state-while-revalidate isn't supported in all browsers: https://caniuse.com/#search=stale-while-revalidate. What is the browser support like for this library?

**Comment by huozhi** [comment, marked as answer]
@jross53 caniuse.com provides the support tables for web standard features (standard API or browser abibilities) across all browsers including desktop and mobile. it can't tell the support for a library since it cannot detect which API/web features that been used in the library you queried.

I think you can check https://github.com/vercel/swr/issues/407#issuecomment-632184527 for details. 
in short terms, swr can run on the browsers that support ES5 syntaxes and `Map` api. which is basically the following

![image](https://user-images.githubusercontent.com/4800338/85586374-7645c680-b673-11ea-9ed0-4c71d642da6f.png)
